File: Iniciante/Python/1010.py
a, b, c = input().split()
d, e, f = input().split()

a,b = int(a), int(b)
c = float(c)

d,e = int(d), int(e)
f = float(f)

print(f"VALOR A PAGAR: R$ {(b*c) + (e*f):.2f}")

File: Iniciante/JavaScript/1010.js
// This line is necessary to read a file to get the user input, is the filepath that Beecrowd uses
// When executing the code locally Remeber to:
//  -> remove the . before /dev and 'txt' after stdin when submitting to beecrowd (or when uploading to github, but that is optional)
var input = require('fs').readFileSync('./dev/stdin.txt', 'utf8');
var lines = input.split('\n');

var values = [];

lines.forEach(l => {
    values.push(l.trim().split(" "));
})

let [a,b,c,d,e,f] = [...values[0].map(Number), ...values[1].map(Number)];

console.log(`VALOR A PAGAR: R$ ${((b*c)+(e*f)).toFixed(2)}`);